ledger-core/insolar/payload/proto_vnvn.r0.proto

```diff
@@ -229,3 +229,27 @@ message VFindCallResponse {
     CallState Status = 23;
     VCallResult CallResult = 24 [(gogoproto.nullable) = true];
 }
+
+message VCachedMemoryRequest {
+    option (insproto.id) = 1013;
+    Reference Object = 21;
+    Reference StateID = 50;
+}
+
+message VCachedMemoryResponse {
+    option (insproto.id) = 1014;
+    enum Status {
+        UnknownCachedMemoryState = 0; // zero value in Go
```

Comment:
I know that you fixed that once, but I suggest to use prefix `CachedMemoryState` (for example `CachedMemoryStateFound`) - for constants defined in `.proto` files.

Reply:
done